salt/metalk8s/kubernetes/cni/calico/deployed.sls

```diff
@@ -635,18 +3667,32 @@ spec:
             - name: IP
               value: "autodetect"
             - name: IP_AUTODETECTION_METHOD
-              # NOTE: Use first CIDR as all calico node should have route to
-              #       all workload CIDR
-              value: can-reach={{ networks.workload_plane.cidr[0].split('/')[0] }}
+              # NOTE: Use all workload CIDRs
+              value: cidr={{ networks.workload_plane.cidr | join(',') }}
```

Comment:
Oh, sweet, is this a new feature @TeddyAndrieux ?

Reply:
Yeppp new in 3.16.0 pretty cool especially in our case